content/governance.md

```diff
@@ -34,19 +34,19 @@ New members may be proposed by any existing member by email to [prometheus-team]
 
 If the new member proposal is accepted, the proposed team member should be contacted privately via email to confirm or deny their acceptance of team membership. This email will also be CC'd to [prometheus-team][team] for record-keeping purposes.
 
-If they choose to accept, the following steps are taken:
-
-* Team members are added to the [GitHub organization][gh] as _Owner_.
-* Team members are added to the [team mailing list][team].
-* Team members are added to the list of team members in this document.
-* New team members are announced on the [developers mailing list][devs] by an existing team member.
+If they choose to accept, the [onboarding](#onboarding) procedure is followed.
 
 Team members may retire at any time by emailing [the team][team].
 
-Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team]. For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team].
+For this vote, the member in question is not eligible to vote, does not count towards the quorum, and may not make any changes to this and related documents any more.
+Multiple members can be removed at the same time, but every vote can only cover one single person.
+For parallel, distinct removal votes every team member retains full voting rights and quorum relevance except their own.
 
 Upon death of a member, their team membership ends automatically.
 
+In case the member leaves, the [offboarding](#offboarding) procedure is applied.
```

Comment:
What if they die? They don't leave in that case, but their membership ends.

Reply:
If the membership ends by automation, they leave, so the procedure applies.

Reply:
Different terminology is used, so it's plausible that these are different things where different rules applies. If you want to make all these things explicit, it should be watertight.

Reply:
Re-worded. Should we also take certified mental degradation into account. Think not.

Reply:
This was a serious question, btw. Still think we don't need to, just went exploring corner cases as "someone in a coma" would mean someone else gaining access to their accounts, passwords, etc.

Reply:
Added verbiage.